docs/vector.rst

```diff
@@ -212,28 +212,28 @@ Methods
     ::
 
         # Taichi-scope
-        a = ti.Vector([1.6, 2.3])
+        a = ti.Vector.field([1.6, 2.3])
```

Comment:
Please don't rely too much on `s/.../.../g`, this is **allocating a vector as local variable**. Please revert all changes about `ti.Vector.field`.

Reply:
Yes, it is so dangerous and I forget this local-global difference.
A revert will be made tomorrow, and I will check these changes manually one by one.